src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let newState = structuredClone(state);
```

Comment:
Думаю це не критично, проте зазначу цей момент, бо різниться з чеклістом.
[CODE STYLE] - don't mutate object or arrays - it will cause unexpected results later on. You should make copy using Object.assign or spread operator